process.md

```diff
@@ -61,126 +62,331 @@ This is for two main reasons:
+contributing their time, thoughts, etc. Community participants are usually users
+who have stopped being anonymous and started being active in project
+discussions.
+
+- Responsibilities:
+  - Must follow the [Tekton CoC](code-of-conduct.md)
+- How users can get involved with the community:
+  - Participating in community discussions ([GitHub, Slack, mailing list, etc](contact.md))
+  - Helping other users
+  - Submitting bug reports
+  - Commenting on issues
+  - Trying out new releases
+  - Attending community events
+
+### Contributor
+
+Description: A Contributor makes direct contributions to the project and adds
+value to it. Contributions need not be code. People at the Contributor level
+may be new contributors, and they can contribute occasionally.
+
+A Contributor must meet the responsibilities of a Community Participant, plus:
+
+- Responsibilities include:
+  - Follow the project [contributing guide](CONTRIBUTING.md)
+- Requirements (one or several of the below):
+  - Report and sometimes resolve issues
+  - Occasionally submit PRs
+  - Contribute to the documentation
+  - Participate in [meetings](working-groups.md)
+  - Answer questions from other community members
+  - Submit feedback on issues and PRs
+  - Test, review, and verify releases and patches
+- Privileges:
+  - Invitations to contributor events
+  - Eligible to become an Organization Member
+
+### Organization Member
+
+Description: An Organization Member is an established contributor who regularly
+participates in the project. Organization Members have privileges in project
+repositories.
+
+An Organization Member must meet the responsibilities and has the requirements
+of a Contributor, plus:
+
+- Responsibilities include:
+  - Continues to contribute regularly, as demonstrated by having at least 15
+    contributions a year, as demonstrated by
+    [the Tekton devstats dashboard](https://tekton.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All).
+- Requirements:
+  - Must have successful contributions to the project, including at least one of
+    the following:
+    - Authored and Reviewed 5 PRs,
+    - Or be endorsed by existing Org Members (e.g. if you are joining a team that is working on Tekton).
+  - Must have 2FA enabled on their GitHub account
+- Privileges:
+  - May give commands to CI/CD automation (e.g. `/ok-to-test`)
+  - May run tests automatically without `/ok-to-test`
+  - May [vote in Tekton governing elections](./governance.md#voter-eligibility)
+  - Can recommend other contributors to become Org Members
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open a PR against
+   [org.yaml](https://github.com/tektoncd/community/blob/main/org/org.yaml),
+   adding your GitHub username to `orgs.tektoncd.members`.
+
+### Collaborator
```

Comment:
nit: collaborator and maintainer seem to map almost exactly to reviewer and approver - could it be more clear to use those terms instead? (np if not)

Reply:
In that case let's at least change collaborator to reviewer - i dont feel super strongly tho so if anyone else does plz chime in ( @afrittoli @vaikas @abayer @dibyom )

I kinda see what you mean about approver v. maintainer so im okay with leaving maintainer as is

Reply:
don't have strong opinions on this. Collaborator/Reviewer both sound good to me.

Reply:
Changed collaborator to reviewer - keeping maintainer as is.